contacts/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+
+<html>
+
+    <head>
+        <title>Chat Room</title>
+    </head>
+
+    <body>
+        
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+        <header>
+            <h1>Chat Room</h1>
+        </header>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+        <nav>
+            <ul>
+                <li><a href="D:\code\mini_project\index.html">Home</a></li>
+                <li><a href="index.html">Contacts</a></li>
```

Comment:
This is not required to be a link, as you are already navigated to this page (no reason to have the option to navigate to the same place again).